### PR TITLE
WFLY-20918 Add idle time-based eviction configuration for distributable HttpSessions, SFSBs, Timers

### PR DESCRIPTION
**Pull request by rhusar**
Proposal for
https://issues.redhat.com/browse/WFLY-20918

WildFly Feature Development Process Issue
https://github.com/wildfly/wildfly-proposals/issues/767

### CODE REVIEW

File: clustering/infinispan/WFLY-20918_Add_max-idle_based_eviction_configuration_for_distributable_HttpSessions_SFSBs_Timers.adoc [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+* Integration tests for each distributed state type (HttpSessions, SFSBs, Timers)
+* Performance tests to ensure eviction processing does not introduce significant overhead
+* Manual testing of edge cases: rapid access patterns preventing eviction, configuration changes at runtime, eviction under load
+* Compatibility testing with existing configurations to ensure backwards compatibility
+* Management model tests verifying the new attributes are properly exposed via CLI and management API
+
+Testing will verify:
+
+* Objects idle longer than the configured max-idle time are evicted
+* Recently accessed objects are not evicted regardless of overall cache size
```
Comment (jasondlee) [inline clustering/infinispan/WFLY-20918_Add_max-idle_based_eviction_configuration_for_distributable_HttpSessions_SFSBs_Timers.adoc]: How does this interact with the count-based eviction?

File: clustering/infinispan/WFLY-20918_Add_max-idle_based_eviction_configuration_for_distributable_HttpSessions_SFSBs_Timers.adoc [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+Specific sections will cover:
+
+* Configuration of max-idle based eviction for each distributed state type
+* Interaction between count-based and idle-based eviction
+* Best practices for choosing between eviction strategies
+* Monitoring and troubleshooting eviction behavior
+
+== Release Note Content
+
+WildFly now supports idle time-based eviction (using the `idle-thresold` attributes) for distributable HttpSessions, Stateful JEB Beans, and JEB Timers.
```
Comment (jasondlee) [inline clustering/infinispan/WFLY-20918_Add_max-idle_based_eviction_configuration_for_distributable_HttpSessions_SFSBs_Timers.adoc]: "EJB"

Comment (rhusar) [reply clustering/infinispan/WFLY-20918_Add_max-idle_based_eviction_configuration_for_distributable_HttpSessions_SFSBs_Timers.adoc]: I will change this back, looks like JEB didn't really catch up :)
